scripts/refresh_guest_languages.pl

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env perl
```

Comment:
I'm on the fence on whether a call to this should be added to guest_language.t, or a seperate test put under 03.scripts/. It wouldn't test much other than to be sure that build_wrapper_for_language isn't irreparably broken - so it might not be worth the file manipulation involved.

Otherwise, this looks good and congrats on getting the image down to CD size.

Reply:
I can achieve in a similar way as `t/04.meadow/lsf.t`.
I create and register a wrapper for a "dummy" language. The wrapper is in fact an `ok()` in disguise, a bit like https://github.com/Ensembl/ensembl-hive/blob/version/2.5/t/04.meadow/fake_bin/bacct#L114-L123 but I would make it return 0/1 upon a certain condition to test both outcomes.
This test would test the interaction of `Bio::EnsEMBL::Hive::GuestProcess` with the wrappers:
- does it call the wrapper with the correct arguments
- I can extend it to test that it detects the wrappers correctly

But then I wouldn't have to test scripts/refresh_guest_languages.pl ?

Reply:
Yeah, that's a pretty sensible way to do it.

Reply:
I have added a unit-test for GuestProcess. Let me know if you think it covers enough of the wrapper interaction (incl. the methods `refresh_guest_languages.pl` calls)